vars/neoDeploy.groovy

```diff
@@ -19,33 +19,104 @@ def call(parameters = [:]) {
             error "Archive cannot be found with parameter archivePath: '${archivePath}'."
         }
 
+        def deployMode = utils.getMandatoryParameter(parameters, 'deployMode', 'MTA')
```

Comment:
Could you please convert the parameter values to lower case. Otherwise it would be different to other steps.